res/themes/light/css/_light.scss

```diff
@@ -262,6 +262,10 @@ $togglesw-off-color: #c1c9d6;
 $togglesw-on-color: $accent-color;
 $togglesw-ball-color: #fff;
 
+// Slider
+$Slider-selection-color: $accent-color;
+$Slider-background-color: #c1c9d6;
```

Comment:
we generally prefer all-lowercase variables

Reply:
Ah, just spoted these change requests. Would it be worth adding this to the linter?

Reply:
yes. We should add a lot of stuff to the linter :D